packages/components/src/tooltip/types.ts

```diff
@@ -16,6 +16,10 @@ export type TooltipProps = {
 	 * **Note**: Accepts only one child element.
 	 */
 	children: React.ReactElement;
+	/**
+	 * Custom class name for the tooltip, in addition to the default `components-tooltip`.
```

Comment:
Internal class names are not part of a components' public APIs and we discourage their usage — let's not mention `.components-tooltip`

```suggestion
	 * Custom class name for the tooltip.
```